metrics/web/lib/common/data/model/build_day_data.dart

```diff
@@ -51,7 +52,9 @@ class BuildDayData extends BuildDay implements DataModel {
       failed: numberOfFailedBuilds ?? 0,
       unknown: numberOfUnknownBuilds ?? 0,
       inProgress: numberOfInProgressBuilds ?? 0,
-      totalDuration: Duration(milliseconds: totalDurationInMilliseconds ?? 0),
+      successfulBuildsDuration: Duration(
+        milliseconds: successfulBuildsDurationMilliseconds ?? 0,
+      ),
```

Comment:
Would it be great to move the `Duration` creation to a variable to be consistent with other fields in this method?